agent/roles/manifests/templates/nmstateconfig_yaml.j2

```diff
@@ -17,19 +17,19 @@ spec:
     {{ net.interfaces(macs[loop.index0]) }}
 {% if ip_stack == "v4" %}
         ipv4:
-          {{ net.ip(ips[loop.index0], cluster_host_prefix_v4) }}
```

Comment:
It would be good to have the change that is cleaning up omissions from #1554 in a separate commit.

Reply:
Yes I will push a separate commit for that